### ISSUE TITLE
Autocomplete, Autoclose and code suggestion not working!

### ISSUE THREAD

**Issue by QuintinS** [Shopify/theme-check-vscode#124]
Hi all, so I've been trying for a few hours to get the autocomplete, autoclose and autosuggestion working, but with no luck.

Oddly, linting seems to be working just fine (I'm getting code highlighting in VSCode when there's a syntax error / suggestion) but no autocompletion features are working at all.

So far I've:

- Updated to MacOS Ventura.
- Uninstalled and reinstalled the Shopify CLI via Homebrew (I'm on version 3)
- Deleted VSCode and all extension/app data. (fresh install)

Here's a gif of what I'm seeing:
![voltage--elate-1680529027080](https://user-images.githubusercontent.com/6787708/229529452-9213cc93-72e6-468b-94a0-288ba5850c02.gif)

Any idea what might be going wrong here?

Thanks in advance!

**Comment by thelornenelson** [comment]
I'm seeing the same thing in 1.9.7 - the tag name is suggested for autocomplete but hitting tab or selecting it from the list only completes the tag name (ie `if`, `liquid`, etc)

<img width="1021" alt="Screen Shot 2023-04-13 at 11 22 32" src="https://user-images.githubusercontent.com/24596583/231850086-19f07d69-5575-4939-8356-608f20a1b44d.png">

**Comment by timdmackey** [comment]
I spent the evening digging around in this extension and the theme-check language server. I've only worked with Textmate-style extensions before, so it took a while to get familiar with language servers and how they work in VS Code. 

Anyhow, it doesn't appear that this extensions has autocomplete support as we've gotten used to with other liquid extensions. It does offer completion for tag and filter _**names**_, as well as object names and properties (cool!), but it won't autocomplete an entire tag or sequence of tags like we're used to. For now, you'll need to keep using a separate extension to provide those autocomplete snippets for you.

I'm going to continue looking at this extension to see if and how it makes sense to write snippet support for it, but until then I'll just reinstall the other liquid extension I was using to get snippet support back.
